### ISSUE TITLE
RStudio 1.1.383 started but could not do anything. Needed to use Task manager to force-quit

### ISSUE THREAD

**Issue by tungmilan** [rstudio/rstudio#1612]
Hello,

I was using the latest RStudio w/ R 3.4.2 on my Windows 10 (64-bit) laptop. After trying knitting [this Rmd doc](https://github.com/LSRU/tv_course/blob/master/lectures/lecture08_purrr.Rmd), I started having problem with RStudio (not sure if they are related though).   

RStudio became very slow and nothing could be done: neither files nor packages could be opened/installed. I couldn't close the IDE either and needed to force-quit it using Task manager. Uninstalled then reinstalled didn't solve the problem. Rgui.exe works normal w/o RStudio.   

Things went back to normal after downgrading to v1.0.153.

[diagnostics-report.txt](https://github.com/rstudio/rstudio/files/1385243/diagnostics-report.txt)

**Comment by jmcphers** [comment]
Saw this error in your logs:

    15 Oct 2017 06:37:23 [rsession-user007] ERROR system error 5 (Access is denied) [path=C:/Users/user007/Documents/.Rhistory]

Are you able to read/write that file successfully outside RStudio?

Does the problem happen in a new project?

**Comment by hugeme** [comment]
@jmcphers  same here. after brew cask install rstudio, I cant used newest rstudio version. initially I assumed its only related to projects, but quickly realized that outside of projects its just stuck if I for instance try to install some package. both rdesktop.log and rsession-user.log in  ~/.rstudio-desktop/log/
are empty.

**Comment by tungmilan** [comment]
@jmcphers I run my laptop as the Admin so reading/writing permission shouldn't be a problem. As I said earlier, things went back to normal after downgrading to v1.0.153. Thanks!

**Comment by jmcphers** [comment]
I cloned the `tv_course` repro and successfully knitted the doc, but wasn't able to reproduce the issue. :-/ 

Was RStudio using any CPU when this happened? If so, does Activity Monitor tell you whether it's the front end (rstudio) or R session (rsession) that's busy?

**Comment by tungmilan** [comment]
Thanks @jmcphers ! I've just uninstalled  v1.0.153 and reinstall v1.1.383 again. It took more than 2 minutes to open a file or display the dialog box when clicking on `Tools > Install Packages`. When I click on `knit` button, nothing happened. I needed to force-quit RStudio again using Task manager as it didn't close by itself after more than 15mins. Note: Avira antivirus was disabled.

Attached are the screenshot and new error log

![rstudio_task_manager](https://user-images.githubusercontent.com/6590141/31834333-7700c48e-b582-11e7-960f-6a5a48a01b4d.png)
![rstudio_task_manager2](https://user-images.githubusercontent.com/6590141/31834432-ca71f78c-b582-11e7-9d90-f60936fbc227.png)
[diagnostics-report.txt](https://github.com/rstudio/rstudio/files/1402819/diagnostics-report.txt)

**Comment by jmcphers** [comment]
Interesting, it doesn't look like it's doing any work. 

Does this happen in a brand new project? 

Does resetting your state help? <https://support.rstudio.com/hc/en-us/articles/200534577-Resetting-RStudio-Desktop-s-State>

**Comment by GegznaV** [comment]
I had a problem with RStudio (in Windows 10): It did not start properly. The state resetting advice did not help. But I deleted this file: `C:\Users\ <user-name> \AppData\Roaming\RStudio\desktop.ini` (Create a backup in case the problem is not there). And the problem was solved. The file contains some RStudio settings.

**Comment by tungmilan** [comment]
@jmcphers & @GegznaV : RStudio v1.1.383 works normal again after deleting `%localappdata%\RStudio-Desktop` & `%AppData%\RStudio` & a clean reinstall

Thanks again for the help guys!

**Comment by kevinushey** [comment]
Any chance you have the old copy of desktop.ini? Would you be willing to share it with us?

**Comment by tungmilan** [comment]
Unfortunately I deleted the desktop.ini file after a day. Wish I saw your post earlier @kevinushey :/

**Comment by kevinushey** [comment]
Dang -- as a last-ditch effort, any chance it's still sitting in your recycle bin?

**Comment by GegznaV** [comment]
I saved this file with a comment "RStudio-backup (breaks RStudio)", but I'm not sure if the problem was really there that time. The contents of _desktop.ini_:

```
[General]
view.zoomLevel=1.4

[mainwindow]
geometry=@ByteArray(\x1\xd9\xd0\xcb\0\x2\0\0\0\0\aw\0\0\0\x10\0\0\r\0\0\0\x4\x13\0\0\b0\0\0\0\x65\0\0\vO\0\0\x2\xd6\0\0\0\x1\x2\0\0\0\x5x)
[.ShellClassInfo]
IconResource=C:\Windows\System32\shell32.dll,77
[ViewState]
Mode=
Vid=
FolderType=Generic
```

**Comment by tungmilan** [comment]
@kevinushey I only keep the %localappdata%\RStudio-Desktop folder unfortunately. Let me know if you want to look at that as well. My desktop.ini is a little bit different from @GegznaV iirc. It only had 2 lines

`[mainwindow]
geometry=@ByteArray(xxx)`

**Comment by kevinushey** [comment]
Interesting; thanks. I tried using that `desktop.ini` on my own RStudio v1.1 instance and it all worked fine. The `geometry` field is just used for saving + restoring the position of the IDE window when it's launched; if that field is corrupt / invalid for some reason we normally just launch a new IDE window centered on the screen.

`view.zoomLevel` also just indicates the zoom level; an invalid zoom level can cause weird things to happen but not a crash on startup.

**Comment by GegznaV** [comment]
I was using a preview version of RStudio at that moment. On startup an errorbox with a message which states that there is a java script error popped up, and most of the RStudio windows were blank and did not responded. I was able to do nothing with RStudio. When I rolled back to v1.0 RStudio, ptoblem disappeared. When I reinstalled  v1.1,  the problem persisted. So I started to search for files on my hard drive, which do not install with RStudio. I found that desktop.ini, deleted it and the problem was solved. 

I faced this problem several times. And I tried to delete the rows below "zooming". It helped too. 

The problem usualy started while I was working in an RStudio project. And at first I was not able to enter some of my projects. Later it became a problem for all RStudio.

I'm not sure if it is the same problem as @tungmilan described. But for him deleting the ini file was a part of solution. As it was for me. It was a solution for some of my students, who faced this problem with preview version of RStudio.
